gandalf.js

```diff
@@ -2,106 +2,97 @@
  * Extras Plugin
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?